src/main/java/gov/usds/case_issues/db/model/NoteAssociation.java

```diff
@@ -10,9 +11,11 @@
 
 	@NaturalId
 	@ManyToOne(optional=false)
+	@JoinColumn(nullable=false)
 	private CaseSnooze snooze;
 	@NaturalId
 	@ManyToOne(optional=false)
+	@JoinColumn(nullable=false)
 	private CaseNote note;
```

Comment:
Do `CaseNote` and `CaseSnooze` not also need the `@NotNull` validation?

Reply:
Yes, and they probably aren't the only ones. Should open up an issue to audit that across the codebase.  (There's a tool that would make that really easy but I haven't ever gotten it to work, so probably grep is my friend.)

Reply:
Opened #37 to track.